morango/utils/uuids.py

```diff
@@ -127,6 +147,7 @@ class InstanceIDModel(UUIDModelMixin):
     database = models.ForeignKey(DatabaseIDModel)
     counter = models.IntegerField(default=0)
     current = models.BooleanField(default=True)
+    db_path = models.CharField(max_length=100)
```

Comment:
May want a slightly longer `max_length` (1000?), since we can't control the paths people are putting their DBs in.